pdns/recursordist/Makefile.am

```diff
@@ -216,3 +222,14 @@ $(MANPAGES):
 	exit 1
 endif
 endif
+
+if !HAVE_LUA_HPP
+# no lua.hpp on solaris :(
+lua.hpp:
```

Comment:
In the odd case that the OS's (or user's with `CFLAGS=-Imyspeciallua`) lua.hpp is 'special' (patched from upstream for whatever reason), we don't break this setup